master/pkg/aproto/master_message.go

```diff
@@ -113,47 +110,39 @@ func (c ContainerStarted) Addresses() []cproto.Address {
 		if info.NetworkSettings == nil {
 			return nil
 		}
-		networks := info.NetworkSettings.Networks
-		ipAddresses := make([]string, 0, len(networks))
-		for _, network := range networks {
-			ipAddresses = append(ipAddresses, network.IPAddress)
+
+		// We used to return a cproto.Address for each network in info.NetworkSettings.Networks that existed, but the
+		// harness only wants the tuple (host_ip, host_port) and disregards container_ip, and (!!) if it does receive
+		// two cproto.Address mappings for the rendezvous port of a container, it would just explode anyway because it
+		// expects a single address per container and doesn't know which one to use.
+		containerIPOnNetwork := "missing"
+		if containerNetwork := info.NetworkSettings.Networks[networkMode.NetworkName()]; containerNetwork != nil {
```

Comment:
to be honest I can't verify this is correct, I'm not saying this is wrong

I'm just not sure if ```info.NetworkSettings.Networks``` always will exist?

and the harness is okay with getting only one address in some cases?

Reply:
nothing ever uses containerIP anyway. i don't suspect it will.